src/components/EntityPage/EntityScorecardsCard.tsx

```diff
@@ -32,24 +39,65 @@ const useStyles = makeStyles<BackstageTheme>(theme => ({
 }));
 
 interface EntityScorecardsCardProps {
-  scores: ServiceScorecardScore[];
+  entityLoading: boolean;
+  scoresLoading: boolean;
+  entity: Entity | undefined;
+  entityError: Error | undefined;
+  scoresError: Error | undefined;
+  scores: ServiceScorecardScore[] | undefined;
   selectedScorecardId?: number;
   onSelect: (scorecardId: number) => void;
 }
 
 export const EntityScorecardsCard = ({
+  entityLoading,
```

Comment:
it's more idiomatic to have the loaders / errors outside of the shared component, since it's not doing the fetch itself -- so we can just have `entity` as a new prop, or even `componentRef` as the new prop

even though it's repeated code it's a better boundary for the component